### DISCUSSION TITLE
How to make Workdone Description optional

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [madrecha/jamku#606] [QA]
Suggested by `Kalpesh Kochar`

In my office, we don't view the workdone description.
To facilitate faster data entry for timesheet, I want to make the Workdone Description optional

**Comment by Divwork92** [comment]
Adding Work done description is optional

Follow the below steps to make work done description optional - 
1. Go to office configuration 
2. Switch to settings tab
3. Under heading "work done" , make 'Work done Description' as NO
4. Click on 'update' on top right to save the changes made

*In case if it still asks to add work done, do a force refresh by pressing ctrl+F5 in a one go
